src/index.html

```diff
@@ -256,12 +259,12 @@ <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">It just takes a few steps
       </section>
 
       <section class="smaller-content-container inner-content-padding-x mxauto flex-col fade-in-up">
-        <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">Expecting more from ourselves and helping inspire our clients.</h3>
+        <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">Expecting more from ourselves, and helping inspire our clients.</h3>
         <p class="off-1 md:off-4 md:col-8 xl:off-6 xl:col-6 text-paragraph mb2_5 atlas">
-          We live and work in New York City, where a new DTC brand is released on what feels like a daily basis. Sanctuary is now after four years in the industry in a fortunate position to be able to say no to potential clients whose mission we don't align with. We're fortunate to work with incredible clients who are moving. 
+          We live and work in New York City, where a new DTC brand is released on what feels like a daily basis. After four years in this industry, we're now in a fortunate position to be able to say turn down potential clients whose mission don't align with ours.
```

Comment:
to be able to say turn down potential clients whose mission
- remove say, make mission plural